### ISSUE TITLE
Stack overflows

### ISSUE THREAD

**Issue by langston-barrett** [langston-barrett/tree-crasher#8]
When fuzzing, I sometimes see this:
```
thread '<unknown>' has overflowed its stack
```
Not sure why.

**Comment by langston-barrett** [comment]
Might have something to do with stack traces like this :sweat_smile: 
```
#0  0x00007ffff7d659ca in _int_malloc ()
   from /nix/store/b2hc0i92l22ir2kavnjn3z5z6mzabbvm-glibc-2.34-210/lib/libc.so.6
#1  0x00007ffff7d66f83 in malloc ()
   from /nix/store/b2hc0i92l22ir2kavnjn3z5z6mzabbvm-glibc-2.34-210/lib/libc.so.6
#2  0x00005555556f5e4a in ts_malloc_default ()
#3  0x0000555555701f0e in ts_tree_cursor_init ()
#4  0x0000555555701f52 in ts_tree_cursor_new ()
#5  0x00005555556bb822 in tree_sitter_edit::editor::Editor::contains_edit ()
#6  0x00005555556bb8ec in tree_sitter_edit::editor::Editor::contains_edit ()
#7  0x00005555556bb8ec in tree_sitter_edit::editor::Editor::contains_edit ()
#8  0x00005555556bb8ec in tree_sitter_edit::editor::Editor::contains_edit ()
#9  0x00005555556bb8ec in tree_sitter_edit::editor::Editor::contains_edit ()
#10 0x00005555556bb8ec in tree_sitter_edit::editor::Editor::contains_edit ()
#11 0x00005555556bb8ec in tree_sitter_edit::editor::Editor::contains_edit ()
#12 0x00005555556bb8ec in tree_sitter_edit::editor::Editor::contains_edit ()
#13 0x00005555556bb8ec in tree_sitter_edit::editor::Editor::contains_edit ()
#14 0x00005555556bb8ec in tree_sitter_edit::editor::Editor::contains_edit ()
#15 0x00005555556bb8ec in tree_sitter_edit::editor::Editor::contains_edit ()
#16 0x00005555556bb8ec in tree_sitter_edit::editor::Editor::contains_edit ()
#17 0x00005555556bb8ec in tree_sitter_edit::editor::Editor::contains_edit ()
#18 0x00005555556bb8ec in tree_sitter_edit::editor::Editor::contains_edit ()
#19 0x00005555556bb8ec in tree_sitter_edit::editor::Editor::contains_edit ()
#20 0x00005555556bb8ec in tree_sitter_edit::editor::Editor::contains_edit ()
#21 0x00005555556bb8ec in tree_sitter_edit::editor::Editor::contains_edit ()
```

**Comment by langston-barrett** [comment]
Fixed in v0.2.1 after some fixes in tree-sitter-edit.
